q2_deblur/plugin_setup.py

```diff
@@ -128,8 +122,37 @@ def denoise(demultiplexed_seqs: SingleLanePerSampleSingleEndFastqDirFmt,
         'jobs_to_start': qiime2.plugin.Int,
         'hashed_feature_ids': qiime2.plugin.Bool
     },
+    parameter_descriptions={
+        'pos_ref_fp': ("Positive (16S) filtering database. Keep all sequences "
+                       "aligning to this FASTA file"),
+        'neg_ref_fp': ("Negative (artifacts) filtering database. Keep all "
```

Comment:
*Negative filtering database. Discard all sequences aligning to these sequences.*

(Note that as is currently written, you're saying *Keep all*.